File: python/day2.py
def switch_case(s: str) -> str:
    out = ''
    for symbol in s:
        if symbol.isalpha():
            if symbol.islower():
                out += symbol.upper()
            else:
                out += symbol.lower()
        else:
            out += symbol
    return out


print(switch_case("hELLO! hOW ARE you?"))

File: R/day2.R
switch_case = function(s){
  out = ""
  for (i in strsplit(s, split = "")[[1]]){
    if (toupper(i) == i){out=paste(out, tolower(i), sep="")}
    else if (tolower(i) == i){out=paste(out, toupper(i), sep="")}
    else out=paste(out, toupper(i), sep="")
  }
  print(out)
}

switch_case("hELLO! hOW ARE you?")